Source/SvgNodeReader.cs

```diff
@@ -139,18 +126,18 @@ private void ParseEntities()
         /// </summary>
         public override void ResolveEntity()
         {
-            if (this.NodeType == XmlNodeType.EntityReference)
+            if (NodeType == XmlNodeType.EntityReference)
             {
-                if (this._entities.ContainsKey(this.Name))
+                if (_entities.ContainsKey(Name))
```

Comment:
One possible fix is to change it to an `Entities` property, but is there a reason not to do so ?

Reply:
It has been a property in the first place, and the bug came from not using it. In this case I see no reason for a property - it only adds complexity instead of removing it (being private, not automatic, and both readable and writable).